source/frontend/src/features/properties/map/acquisition/detail/models.ts

```diff
@@ -19,9 +24,9 @@ export class DetailAcquisitionFile {
       model?.acquisitionPhysFileStatusTypeCode?.description;
     detail.acquisitionTypeDescription = model?.acquisitionTypeCode?.description;
     detail.regionDescription = model?.regionCode?.description;
-    // acquisition team array
     detail.acquisitionTeam =
       model?.acquisitionTeam?.map(x => DetailAcquisitionFilePerson.fromApi(x)) || [];
+    detail.acquisitionOwners = model?.acquisitionFileOwners || [];
```

Comment:
is this right? I would expect a conversion here.

Reply:
Removed